### PR TITLE
CTPPS: exceptions fix

### PR DESCRIPTION
**Pull request by jan-kaspar**
- geometry: no (misleading) error is printed when no alignment corrections are loaded, LogVerbatim is used instead of printf and cout
- DQM: histograms created with fixed ranges to avoid problems when merging histograms from several module instances

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc [hunk 1]
```diff
@@ -416,33 +416,34 @@ void TotemRPGeometryESModule::ApplyAlignments(const edm::ESHandle<DDCompactView>
...
-                << alignments->GetSensorMap().size() << " sensor alignments applied.\n";
-        }
-    } else {
-        if (verbosity)
-            cout << ">> TotemRPGeometryESModule::produceMeasuredDDCV > Measured geometry: No alignments applied.\n";
+  // load alignments
+  edm::ESHandle<RPAlignmentCorrectionsData> alignments;
+  try {
+    iRecord.getRecord<RPMeasuredAlignmentRecord>().get(alignments);
+  } catch (...) {}
```
Comment (ianna) [inline Geometry/VeryForwardGeometryBuilder/plugins/TotemRPGeometryESModule.cc]: @jan-kaspar - please, do not use catch(...). It's against CMS coding rules.

**Comment by jan-kaspar** [comment]
@ianna Although it may look like, I'm not adding the `catch`, it was in the code before. And it was there since in our workflow we allow not to insert any alignment corrections. In my patch I only remove the error message which was misleading since there is nothing wrong when no alignments are supplied.

**Comment by dmitrijus** [comment]
+1

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**